README.md

```diff
@@ -2,6 +2,8 @@
 
 Try it live: [https://manifest-validator.appspot.com/](https://manifest-validator.appspot.com/)
 
+Tip: You can prepopulate a website URL with [https://manifest-validator.appspot.com/#https://example.com](https://manifest-validator.appspot.com/#https://example.com)
```

Comment:
Yes, I get #36 - but the text here needs to be clarified. It doesn't help users who are looking at the README. 

> On 7 Jun 2016, at 1:59 PM, François Beaufort <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -2,6 +2,8 @@
> >  
> >  Try it live: [https://manifest-validator.appspot.com/](https://manifest-validator.appspot.com/)
> >  
> > +Tip: You can prepopulate a website URL with [https://manifest-validator.appspot.com/#https://example.com](https://manifest-validator.appspot.com/#https://example.com)
> See #36
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>